OutOfSchool/OutOfSchool.WebApi/Controllers/V2/WorkshopController.cs

```diff
@@ -164,18 +163,22 @@ public async Task<IActionResult> GetByFilter([FromQuery] WorkshopFilter filter)
         [Consumes("multipart/form-data")]
         public async Task<IActionResult> Create([FromForm] WorkshopCreationDto dto) // TODO: validate by request size
         {
-            // TODO: Also should check for AddressDto == null
             if (!ModelState.IsValid)
             {
                 return BadRequest(ModelState);
             }
 
+            if (dto.Address == null)
```

Comment:
Maybe add a `required` attribute to address field in model and validate it during model validation?
or it doesn't work with form data?

Reply:
I hadn't been sure that we use dto.Address in all situations as required before your wrote this. I will change it on attribute)